logs_analyzer/chapter1/java8/src/main/java/com/databricks/apps/logs/chapter1/LogAnalyzer.java

```diff
@@ -1,34 +1,32 @@
 package com.databricks.apps.logs.chapter1;
 
 import com.databricks.apps.logs.ApacheAccessLog;
-
+import java.io.Serializable;
```

Comment:
This is just how my IDE organizes imports. If it is harmful, then I can revert it.